src/Header/AbstractRecipient.php

```diff
@@ -57,4 +57,26 @@
             ])
         );
     }
+
+    /**
+     * @param array $list pairs of email address and name
+     * @return AbstractRecipient
+     */
+    final public static function fromList(array $list): AbstractRecipient
+    {
+        return new static(
+            new AddressList(
+                array_map(
+                    function ($pair) {
+                        [$emailAddress, $name] = $pair;
```

Comment:
Please add a check here if `$pair` contains 2 elements. What if only one is given?

Reply:
I noticed that we should concern email address without name. The following argument should be valid.

```php
[['bob@example.com', 'Bob'], ['noname@example.com']]
```

Alternatively, I need your opinion; Do we allow passing string `'noname@example.com'` 
 instead of array `['noname@example.com']`?

```php
[['bob@example.com', 'Bob'], 'noname@example.com']
```

Reply:
I rather see only `[['bob@example.com', 'Bob'], ['noname@example.com']]`. I am kind of the strict guy. But I guessed you noticed that by using this library.

Reply:
Yeah, this library is strictly typed 😄